src/_data/sidenav.yml

```diff
@@ -192,6 +192,8 @@
       permalink: /docs/resources/bootstrap-into-dart
     - title: Inside Flutter
       permalink: /docs/resources/inside-flutter
+    - title: Platform Adaptations
```

Comment:
We use sentence case for titles and headings.  Platform Adaptations => Platform adaptations.

Reply:
Done